aula-pratica-03/ex2.c

```diff
@@ -0,0 +1,25 @@
+#include<stdio.h>
+int bin_to_dec(int *vetor);
+
+int bin_to_dec(int *vetor) {
```

Comment:
well, the exercice description dictates this signature, tho. not that this is wrong :man_shrugging: 
```suggestion
int transforma_binario_decimal(int vetor[5]) {
```